src/app/features/events/components/event-card/event-card.component.html

```diff
@@ -1,60 +1,113 @@
+                </app-button>
+
+                <app-button class="event-decline"
+                            [ngClass]="model?.confirmed ? 'disabled' : ''"
+                            [matButtonType]="materialButtonType.Icon"
+                            [tooltip]="(model?.declined ? translations.eventCard.decline.tooltip_off : translations.eventCard.decline.tooltip_on) | translate"
+                            [ariaLabel]="(model?.declined ? translations.eventCard.decline.aria_off : translations.eventCard.decline.aria_on) | translate"
+                            [disabled]="loading || model?.canceled == true || isPast()"
+                            (click)="declineEvent()">
+                    <i *ngIf="!loading" 
+                    class="fa-circle-xmark"
+                    [ngClass]="model?.declined || model?.confirmed ? 'fa-solid' : 'fa-regular'">
+                    </i>
+                    <ngx-skeleton-loader *ngIf="loading" count="1" appearance="circle" animation="progress"></ngx-skeleton-loader>
+                </app-button>
+            </div>
+        </mat-card-content>
+    </mat-card>
+</ng-container>
+
+<!-- Large View -->
+<ng-container *ngIf="view !== 'small'" #templateLarge>
+    <mat-card appearance="outlined" class="event-card-large">
+        <mat-card-header>
+            <div class="event-image-wrapper">
+               <img *ngIf="!loading" [src]="model?.displayPicture" [ngClass]="{'greyed-image-wrapper': model?.canceled || isPast()}">
+               <ngx-skeleton-loader *ngIf="loading" count="1" appearance="line" animation="progress"></ngx-skeleton-loader>
+           </div> 
+       </mat-card-header>
+        <mat-card-content>
+            <div class="event-calendar">
+                <app-calendar-button [model]="model?.startDate" 
+                                    [tooltip]="translations.eventCard.calendar.tooltip | translate"
+                                    [aria]="translations.eventCard.calendar.aria | translate"
+                                    [tooltipDirection]="tooltipDirection.Above"
+                                    [loading]="loading"
+                                    [ngClass]="loading? 'calendar-loading' : ''">
+                </app-calendar-button>
             </div>
+            <div class="event-content" 
+                [ngClass]="model?.canceled == true ? 'canceled' : isPast() ? 'past' : ''"
+                routerLink="{{ '/' + routes.Events  + '/' + model?.id }}">
+                <div class="event-text">
+                    <div *ngIf="!loading" class="title-larger-text">
+                        <div *ngIf="model?.canceled">
+                        </div>
+                        <div>
+                            <span class="canceled" *ngIf="model?.canceled">{{ translations.eventCard.canceled | translate }} </span>{{ model?.title }}
```

Comment:
Canceled should be upppercase. You can add the uppercase pipe like this `translations.eventCard.canceled | translate | uppercase`